### DISCUSSION TITLE
Tech Issue

### DISCUSSION THREAD

**Discussion by ghost** [4ian/GDevelop#4757] [Archive] [answered]
how can i contact the team privateley?
I am experiencing an issue with the core engine and i have to contact the team privateley

**Comment by arthuro555** [comment, marked as answer]
hello@gdevelop.io

**Reply by ghost** [reply]
Alredy written at September
no reponse recived

**Reply by ghost** [reply]
Can anyone help

**Reply by Bouh** [reply]
- The team does not respond to requests for bug reports by email. A bug report must be published on the Issue tab here on Github.
And if there is a need to do so privately=> [hello@gdevelop.io](mailto:hello@gdevelop.io) is open but please send as much information as possible, we need to understand the context. 

- If you need help to make a game we invite everyone to open a topic in the community forum. The team does not provide specific help for creating a game.

**Reply by arthuro555** [reply]
No need to answer, that user has been deleted :/
